include/gevolution/basic_types.hpp

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include "gevolution/config.h"
+#include "LATfield2.hpp"
+
+namespace gevolution
+{
+    using LATfield2::Real;
+    typedef LATfield2::Imag Cplx;
+    
+    // type for the serialization of particles into Gadget snapshots
+    // TODO: enforce this at compile time of the library of leave it as a template?
+    typedef float position_type;
```

Comment:
Serialization of gadget snapshots is messy. There is no good solution to this problem. However it is good practice to specify the types used in a place easy to find.

Reply:
done b343396991509869ef6898bd3bbc3031fc802710